repos/exporter/lib/handler.ts

```diff
@@ -89,6 +94,13 @@ export const handler: Handler<ExportLambdaBody> = async ({ paths, noc, exportPre
                 ? await getTimeRestrictionsByIdAndNoc(ticketWithIds.timeRestriction.id, noc)
                 : [];
 
+            const cap =
+                ['single', 'return', 'flatFare'].includes(ticketWithIds.type) &&
+                'cap' in ticketWithIds &&
+                ticketWithIds.cap
+                    ? await getCapByNocAndId(noc, ticketWithIds.cap.id)
+                    : undefined;
```

Comment:
We dont need to check the fareType here, because we can trust the site that if there is a cap on the ticket, it should be there. And also in the future, we might change it to be for more than singles, returns and flat fares.

I think we can just do 
```suggestion
            const cap = !!ticketWithIds.cap ? await getCapByNocAndId(noc, ticketWithIds.cap.id) : undefined;
```